apps/api/tests/test_guest.py

```diff
@@ -109,9 +109,7 @@ def test_successful_guest_verification_provides_identity(
 
 
 @patch("auth.guest_auth._get_existing_key", autospec=True)
-@patch("auth.guest_auth._remove_guest_key", autospec=True)
 def test_invalid_guest_verification_is_unauthorized(
-    mock_remove_guest_key: AsyncMock,
     mock_get_existing_key: AsyncMock,
```

Comment:
Nitpick: also missed on #148, but for consistency, it would make more sense for `mock_get_existing_key` to be the first parameter (and switch the order of the patches). Refer to `test_successful_guest_verification_provides_identity` for the suggested order.